index.html

```diff
@@ -364,5 +364,16 @@
 in terms of improved access to government services, banking, healthcare and other critical services.
 However, the resulting harms are not trivially justified by those benefits.
 
+## Conclusion
+
+The TAG supports the work on digital credentials. 
+And we also encourage the groups working in this space to consider all the 
+risks associated with these technologies with open eyes, and to develop mitigations
+where possible. We encourage threat modelling to understand how these technologies
+can be misused and potentially lead to unintended and negative societal consequences.
+In the [Ethical Web Principles](https://www.w3.org/TR/ethical-web-principles/#intro), we called
```

Comment:
Does this link to the intro specifically for a reason (or could we use `[[ETHICAL-WEB]]` as a citation instead?)

Reply:
The Intro is where the quote is pulled from, but I'm happy to just change it to a citation if you think that's better.